rasa_core/server.py

```diff
@@ -868,6 +869,83 @@ async def retrieve_conversation_messages(request: Request, sender_id: Text):
 
     return app
 
+    @app.route("/conversations/trigger_action_to_ids_24h", methods=['POST'])
```

Comment:
Although, this seems fine as endpoint name. But, to be more consistent with the functionality of this endpoint, I would choose something which gives us a hint about that this is meant for monitoring and resume dead conversations. For instance, /conversations/monitor-dead-conversations or /conversations/resume-dead-conversations.